baseapp/msg_service_router.go

```diff
@@ -27,10 +27,11 @@ type MessageRouter interface {
 
 // MsgServiceRouter routes fully-qualified Msg service methods to their handler.
 type MsgServiceRouter struct {
-	interfaceRegistry codectypes.InterfaceRegistry
-	routes            map[string]MsgServiceHandler
-	hybridHandlers    map[string]func(ctx context.Context, req, resp protoiface.MessageV1) error
-	circuitBreaker    CircuitBreaker
+	interfaceRegistry    codectypes.InterfaceRegistry
```

Comment:
We write a test that demonstrates how this new option works:

- events are emitted when the field is false
- events are not emitted when true

by running some SDK message on the router